.vscode/settings.json

```diff
@@ -6,5 +6,25 @@
       ],
       "url": "http://json-schema.org/draft-07/schema#"
     }
-  ]
-}
\ No newline at end of file
+  ],
+  "yaml.schemas": {
```

Comment:
I would use a `data/` prefix for values, so as to pinpoint factory data.

Reply:
Added `data/**/` prefix, so we can have `data/<stage-name>/<object_class>` structure when pointing to directory in `factories_config`.

Reply:
`stage-name` is not needed, paths will always be `data/object-class` for both modules and FAST :)

Reply:
well I lied, I point FAST factories for my org to my own data folder which is structured exactly like you wrote 😊, so we should support both `data/object-class` and `data/stage/object-class`

Reply:
both works with `data/**/`

Reply:
yep, absolutely that was the intention of my last message